.github/ISSUE_TEMPLATE/regular-issue.yaml

```diff
@@ -0,0 +1,47 @@
+name: Generic Issue
+description: Open a generic Issue for the OpenJS CPC
+labels:
+  - cross-project-council-agenda
```

Comment:
We don't necessarily want all issues to be brought to the agenda.

Reply:
I see that eventually each open issue gets added to the agenda, we can of course always remove the label if we don't need it, but I often see it more being added to issues 🤔 

I'm fine by removing a default label to be honest :)